gui/builtinViews/fittingView.py

```diff
@@ -339,7 +339,11 @@ def removeItem(self, event):
     def removeModule(self, module):
         sFit = service.Fit.getInstance()
         fit = sFit.getFit(self.activeFitID)
-        populate = sFit.removeModule(self.activeFitID, fit.modules.index(module))
+        try:
+            populate = sFit.removeModule(self.activeFitID, fit.modules.index(module))
+        except ValueError:
+            # This module isn't in our list of modules, don't remove anything. Likely a special snowflake.
+            populate = None
```

Comment:
Okay, so we no longer show the remove option in the menu.  So used to seeing it, that it looked normal with it there and weird without it.

We still need this try/catch though, because of double click to remove.  Like I said, when it exceptions because of this it fails in the way we want it to (keeps going and doesn't remove anything), but I think it's cleaner to catch the exception and handle it.

I can add debug logging if we want to record these scenarios.